examples/intkey_rust/Cargo.toml

```diff
@@ -19,6 +19,12 @@
 name = "sawtooth-intkey-tp-rust"
 version = "0.5.3"
 authors = ["Bitwise IO, Inc."]
+edition = "2018"
```

Comment:
This isn’t needed, or if it is, the default value is not 2018.

Reply:
ok! I just copied it from the xo example. Should it be removed from there too? https://github.com/hyperledger/sawtooth-sdk-rust/blob/main/examples/xo_rust/Cargo.toml#L20